docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+services:
+  pdam:
+    image: git.hack13.dev/hack13/personal-digital-asset-manager:latest
+    restart: unless-stopped
+    volumes:
+      - ./uploads:/app/static/uploads # Not required for S3 Storage Back-end
+      - ./app.db:/app/instance/app.db
+    environment:
+    ## Database Configuration
+    #  - DATABASE_URL=sqlite:///instance/app.db
+
+    ## File Storage
+    #  - STORAGE_URL=file://static/uploads

```

Comment:
Suggest making a small change here for file://app/static/uploads... because right now it will not work with that one.